dotcom-rendering/src/web/components/CommentCount.test.tsx

```diff
@@ -6,16 +6,17 @@ import { CommentCount } from './CommentCount';
 import { decidePalette } from '../lib/decidePalette';
 
 describe('CommentCount', () => {
+	const format = {
+		theme: ArticlePillar.News,
+		design: ArticleDesign.Standard,
+		display: ArticleDisplay.Standard,
+	};
```

Comment:
I think this abstraction is fair although personally I would still keep this code inline. I guess the thing to consider is what do you need from this code? What is more important:

1) Fewer lines of code
2) Lack of repetition
3) Easy to change in future

My personal view is that we should be writing code that prioritises ease of maintenance in the future. One of our key goals with DCR is around building something that will endure.

All that said, this particular abstraction is fine and I'm totally happy to keep this code. I just thought this was a nice opportunity to raise this discussion as I think it's an important concept

Reply:
I get your point, thanks for sharing. My reason was that the the same format was being repeated several times and thought to abstract it to reduce repetition. I think here maybe I can improve the ease of code reading by giving it a better name such as `standardNewsFormat` rather than just `format`